content/en/docs/tasks/administer-cluster/cluster-upgrade.md

```diff
@@ -59,6 +59,11 @@ For each node in your cluster, [drain](/docs/tasks/administer-cluster/safely-dra
 that node and then either replace it with a new node that uses the {{< skew currentVersion >}}
 kubelet, or upgrade the kubelet on that node and bring the node back into service.
 
+{{< warning >}}
+Draining nodes before upgrading kubelet ensures that pods are re-admitted and containers are
+re-created which may be necessary to resolve some CVEs or bugs.
+{{</ warning >}}
```

Comment:
I would probably use a caution, not a warning. Warnings are for “get this right or you **will** put your cluster at risk” and we use them sparingly.

Reply:
Try this (or the `caution` equivalent):
```suggestion
{{< warning >}}
Draining nodes before upgrading kubelet ensures that pods are re-admitted and containers are
re-created, which may be necessary to resolve some security issues or other important bugs.
{{</ warning >}}
```

Reply:
Caution sounds good to me, reworded and changed.